### PR TITLE
Initial ETL Application and Plugin Changes

### PR DESCRIPTION
**Pull request by awholegunch**
This is a first PR (of two) to update the ETL Applications. This is the "outer" portion, and does not change the existing plugins, except to move their location. The plugins themselves will be updated in the next one.

This was split into two because it was getting large, some fixes were needed in the build, and these changes should go in so other PRs can use the new references if required.

### CODE REVIEW

File: cdap-docs/included-applications/source/etl/creating.rst [hunk 1]
```diff
@@ -57,30 +57,30 @@ configuration for a Batch Adapter that runs every minute, reading data from a St
...
-- Using the :ref:`RESTful API <http-restful-api-lifecycle-create-app>`::
+- Using the :ref:`Lifecycle RESTful API <http-restful-api-lifecycle-create-app>`::
 
-    PUT /v3/namespaces/default/adapters/streamAdapter -d @config.json 
+    PUT /v3/namespaces/default/apps/streamETLApp -d @config.json 
 
 - Using :ref:`CDAP CLI <cli>`::
 
-    $ create adapter streamAdapter <path-to-config.json>
+    $ create app streamETLApp <path-to-config.json>
```
Comment (albertshau) [inline cdap-docs/included-applications/source/etl/creating.rst]: create app streamETLApp cdap-etl-batch |version| system <path-to-config.json>

File: cdap-docs/included-applications/source/etl/creating.rst [hunk 2]
```diff
@@ -115,20 +115,20 @@ Stream after performing a projection transformation::
...
 
 
-The ETL Real-Time Template expects an instance property that will create *N* instances of the
-adapter that run concurrently. In Standalone CDAP mode, this is implemented as multiple threads;
-while in Distributed CDAP mode, it will create different YARN containers. The
-number of instances of a real-time adapter cannot be changed during runtime.
+An ETL Real-Time Application expects an instance property that will create *N* instances
+of the worker that run concurrently. In Standalone CDAP mode, this is implemented as
+multiple threads; in Distributed CDAP mode, it will create different YARN containers. The
+number of worker instances of a real-time application cannot be changed during runtime.
```
Comment (albertshau) [inline cdap-docs/included-applications/source/etl/creating.rst]: If the number of instances needs to be changed, the worker must be stopped, then the application configuration must be updated to the new number of instances.

**Comment by albertshau** [comment]
a couple suggestions for adding a bit more information. Otherwise lgtm

**Comment by albertshau** [comment]
The configs also need to be changed, but I can do that in a separate PR

File: cdap-docs/included-applications/source/etl/creating.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/included-applications/source/etl/creating.rst]: fixed.

File: cdap-docs/included-applications/source/etl/creating.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/included-applications/source/etl/creating.rst]: fixed.

**Pull request merged**